it/controllers/expenses/OtherEquipmentAmountControllerISpec.scala

```diff
@@ -0,0 +1,530 @@
+          dropExpensesDB()
+          userDataStub(userData(fullEmploymentsModel(hmrcExpenses = Some(employmentExpenses(fullExpenses)))), nino, taxYearEOY)
+          insertExpensesCyaData(expensesUserData(isPrior = false, hasPriorExpenses = false,
+            fullExpensesCYAModel), userRequest)
+          authoriseAgentOrIndividual(user.isAgent)
+          urlPost(employmentExpensesAmountPageUrl(taxYearEOY), body = form, follow = false, welsh = user.isWelsh,
+            headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY)))
+        }
+
+        "redirects to the check your details page" in {
+          result.status shouldBe SEE_OTHER
+          result.header("location") shouldBe  Some(CheckEmploymentExpensesController.show(taxYearEOY).url)
+          lazy val cyaModel = findExpensesCyaData(taxYearEOY, userRequest).get
+
+          cyaModel.expensesCya.expenses.claimingEmploymentExpenses shouldBe true
+          cyaModel.expensesCya.expenses.jobExpensesQuestion shouldBe Some(true)
+          cyaModel.expensesCya.expenses.flatRateJobExpensesQuestion shouldBe Some(true)
+          cyaModel.expensesCya.expenses.otherAndCapitalAllowancesQuestion shouldBe Some(true)
+          cyaModel.expensesCya.expenses.businessTravelCosts shouldBe Some(100.00)
+          cyaModel.expensesCya.expenses.jobExpenses shouldBe Some(200.00)
+          cyaModel.expensesCya.expenses.flatRateJobExpenses shouldBe Some(300.00)
+          cyaModel.expensesCya.expenses.professionalSubscriptions shouldBe Some(400.00)
+          cyaModel.expensesCya.expenses.hotelAndMealExpenses shouldBe Some(500.00)
+          cyaModel.expensesCya.expenses.otherAndCapitalAllowances shouldBe Some(newAmount)
+          cyaModel.expensesCya.expenses.vehicleExpenses shouldBe Some(700.00)
+          cyaModel.expensesCya.expenses.mileageAllowanceRelief shouldBe Some(800.00)
+        }
+
+      }
+
+      "redirect to 'check your expenses' page when a prior submission and update otherAndCapitalAllowances to the new amount" which {
+
+        lazy val form: Map[String, String] = Map(AmountForm.amount -> newAmount.toString())
+
+        lazy val result: WSResponse = {
+          dropExpensesDB()
+          userDataStub(userData(fullEmploymentsModel(hmrcExpenses = Some(employmentExpenses(fullExpenses)))), nino, taxYearEOY)
+          insertExpensesCyaData(expensesUserData(isPrior = true, hasPriorExpenses = true,
+            fullExpensesCYAModel), userRequest)
+          authoriseAgentOrIndividual(user.isAgent)
+          urlPost(employmentExpensesAmountPageUrl(taxYearEOY), body = form, follow = false, welsh = user.isWelsh,
+            headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY)))
+        }
+
+        "redirects to the check your details page" in {
+          result.status shouldBe SEE_OTHER
+
+          result.header("location") shouldBe Some(controllers.employment.routes.CheckEmploymentExpensesController.show(taxYearEOY).url)
```

Comment:
you can remove the `controllers.employment.routes` part since you've already imported the route into this file